pkg/sources/postgres/store_test.go

```diff
@@ -51,3 +52,31 @@ func TestStoreTestSuite(t *testing.T) {
 func (s StoreTestSuite) SetupTest() {
 	s.store.db.MustExec("TRUNCATE dog")
 }
+
+func setup(t *testing.T) (*Store, *clock.Mock, context.Context, func(t *testing.T)) {
+	t.Helper()
+
+	config := viper.New()
+	config.AutomaticEnv()
+
+	dbConfig := DBConfig{
+		Host:     config.GetString(appconfig.DBHostConfigKey),
+		Port:     config.GetString(appconfig.DBPortConfigKey),
+		Database: config.GetString(appconfig.DBNameConfigKey),
+		Username: config.GetString(appconfig.DBUsernameConfigKey),
+		Password: config.GetString(appconfig.DBPasswordConfigKey),
+		SSLMode:  config.GetString(appconfig.DBSSLModeConfigKey),
+	}
+	store, err := NewStore(dbConfig)
+	if err != nil {
+		fmt.Printf("Failed to get new database: %v", err)
```

Comment:
Could/should this be `t.Logf` ?

Reply:
Good catch... Actually these two lines could be a `t.Fatalf(...)`!

Reply:
TIL that `t.Fail` and `t.Errorf` allow the test to continue while `t.FailNow` and `t.Fatalf` do not.

By coincidence I was doing some testing prior to your change and my database wasn't running and I got a panic because after that `t.Fail` the test continued to run and try to reference the `store` a few lines below.

Reply:
Yes, and... I get the same problem w/the `panic`, but that is true even on the main branch, so I wasn't gonna focus on that in this PR because I was trying to demonstrate orthogonal to that issue.